new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMap.java

```diff
@@ -321,7 +321,7 @@ public void testCreateMiiConfigMapDomain() {
       checkServiceCreated(managedServerPrefix + i, domainNamespace);
     }
 
-    int adminServiceNodePort = getAdminServiceNodePort(adminServerPodName + "-external", null, domainNamespace);
+    int adminServiceNodePort = getServiceNodePort(domainNamespace, adminServerPodName + "-external", "default");
```

Comment:
is default the namespace? if so, seems like it should be domainNamespace?
if it is the name of the port, then that's ok, though be aware that when istio is enabled in the cluster, the port name will not be default, but probably istio-http or something like that

Reply:
default is the channel name.  Got it that it needs to be different when istio enabled.